core/src/qos_service.rs

```diff
@@ -126,22 +126,22 @@ impl QosService {
         &self,
         transactions: impl Iterator<Item = &'a SanitizedTransaction>,
         transactions_costs: impl Iterator<Item = &'a TransactionCost>,
-        bank: &Arc<Bank>,
+        slot: Slot,
+        cost_tracker: &mut RwLockWriteGuard<CostTracker>,
```

Comment:
could you pass in &mut CostTracker and have this still work?

Reply:
idk if method needs anything related to RwLockWriteGuard

Reply:
i guess i didn't quite understand how the RwLock works

my goal here is to be able to hold the lock on the cost_tracker over multiple function calls

are you saying if i pass this into the function it will still hold the write lock as intended?
`let mut cost_tracker = bank_start.working_bank.write_cost_tracker().unwrap().deref_mut()`

Reply:
talked offline, should be able to take a `&mut CostTracker` and not have this method worry about whether its a mutex, rwlock, or CostTracker w/ no synchronization